### PR TITLE
1105

### PR DESCRIPTION
**Pull request by somlyody**

### CODE REVIEW

**Review by SevenIndirecto — CHANGES_REQUESTED** [review]

File: src/hub/i18n/en.js [hunk 1]
```diff
@@ -45,5 +45,12 @@ export default {
     leo: 'Leo',
     jane: 'Jane',
 	flamma: 'Flamma',
+    kangaroo: 'Kangaroo'
+    first policeman: 'First policeman'
```
[the remark this answers is not in the document]
Comment (SevenIndirecto) [reply src/hub/i18n/en.js:49]: Also even if we added the characters like this, they can't have spaces for 'keys'. So it needs to be something like:
```
first_policeman: 'First policeman'
```

[the remark this answers is not in the document]
Comment (somlyody) [reply src/hub/i18n/en.js:49]: hello, these have their own dialog portraits but they appear only in the Logi-city stories. which are the generic characters?

[the remark this answers is not in the document]
Comment (SevenIndirecto) [reply src/hub/i18n/en.js:49]: Hey, I guess this is fine, just please change to using keys without spaces: ('keys' are the values on the left side of the `:`)

```
firstPoliceman: 'First policeman'
```

**Comment by rmoschetti** [comment]
Since there are some conflicts, and also some new changes in the branch, I'll close this and reopen a new one with all the corrections.

**Pull request closed without merging**